app/models/committee_decision.rb

```diff
@@ -74,6 +74,14 @@ def current_review
     reviews.order(:created_at).last
   end
 
+  def all_details_present?
```

Comment:
This method could be shortened by having the fields in an array and calling `all?` on the array but it does exactly the same thing